mobile/app/src/debug/res/values/app_name.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">IITC Mobile Test</string>
+    <string name="app_name">IITC Mobile</string>
```

Comment:
This change is not necessary because the debug version should have the "Test" postfix

Reply:
Is a relict so i can install official version and this test side by side

Reply:
I don't understand what the question is. The release version is "IITC Mobile", the beta version is "IITC Mobile Beta", the builds from PR or in debug mode are "IITC Mobile Test". These are three different versions that you can have installed at the same time.
In any case, you can choose any name to build on your own. But please remove this line in PR